pythran/pythonic/numpy/stack.hpp

```diff
@@ -34,7 +38,8 @@ namespace numpy
     // Create a new empty list.
     types::list<types::ndarray<
         typename ArraySequence::value_type::dtype,
-        types::array<long, ArraySequence::value_type::value + 1>>> bi(0);
+        types::array<long, ArraySequence::value_type::value + 1>>>
```

Comment:
useless change?

Reply:
That's clang-format

Reply:
Unfortunately not, see the validation... you need to run `clang-format-3.6`, sorry about that